ObjectiveSee/DoNotDisturb.pkg.recipe

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Description</key>
+    <string>Downloads, unzips, verifies and packages the installer of the latest version of DoNotDisturb by Objective-See.</string>
+    <key>Identifier</key>
+    <string>com.github.its-unibas.pkg.DoNotDisturb</string>
+    <key>MinimumVersion</key>
+    <string>0.6.0</string>
+    <key>Input</key>
+    <dict>
+        <key>NAME</key>
+        <string>Do Not Disturb</string>
+    </dict>
+    <key>ParentRecipe</key>
+    <string>com.github.its-unibas.download.DoNotDisturb</string>
+    <key>Process</key>
+    <array>
+        <dict>
+            <key>Arguments</key>
+            <dict>
+                <key>pkgdirs</key>
+                <dict>
+                    <key>usr/local/var/dnd</key>
+                    <string>0775</string>
+                </dict>
+                <key>pkgroot</key>
+                <string>%RECIPE_CACHE_DIR%/%NAME%/pkg</string>
+            </dict>
+            <key>Processor</key>
+            <string>PkgRootCreator</string>
+        </dict>
+        <dict>
+            <key>Processor</key>
+            <string>FileMover</string>
+            <key>Arguments</key>
+            <dict>
+                <key>source</key>
+                <string>%RECIPE_CACHE_DIR%/%NAME%/%NAME% Installer.app</string>
```

Comment:
Are we making a pkg for the `installer.app`? From the name of the recipe I'd assume that the pkg would install DND. 
Since I don't use the .pkg recipe I would be okay with deleting it and not adding it into this pr.